libs/eo/dashboard/shell/src/lib/eo-dashboard-shell.component.ts

```diff
@@ -0,0 +1,82 @@
+/**
+ * @license
+ * Copyright 2020 Energinet DataHub A/S
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License2");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {
+  ChangeDetectionStrategy,
+  Component,
+  NgModule,
+  ViewEncapsulation,
+} from '@angular/core';
+
+import { EoInlineMessageScam } from '@energinet-datahub/eo/shared/atomic-design/ui-atoms';
+import { EoTitle } from '@energinet-datahub/ett/shared/util-browser';
+
+const selector = 'eo-dashboard-shell';
+
+@Component({
+  selector,
+  styles: [
+    `
+      @use '@energinet-datahub/watt/utils' as watt;
+      ${selector} {
+        display: block;
+      }
+      .${selector}__p {
+        @include watt.typography-watt-text-m; // This overrides the styles applied from Angular Material on p tags
+      }
+      .${selector}__eo-inline-message {
+        margin-bottom: var(--watt-space-l);
+      }
+    `,
+  ],
+  template: `
+    <eo-inline-message
+      icon="/assets/images/icons/primary-info-icon.svg"
```

Comment:
suggestion: Use the Watt Icon componetn and add this SVG icon to the `MatIconRegistry`.